README.md

```diff
@@ -5,12 +5,19 @@
 # Tesla Fleet Telemetry
 ---------------------------------
 
-Fleet Telemetry is a simple, scalable, and secure data exchange for device fleets.
+At Tesla we believe that security and privacy should be in the center of any modern technology. We want each and everyone of our customers to decide for themselves what data they share with third parties, how they share it and when it can be shared. We've developped a decentralized framework that allows customers to create a secure and direct brige from their Tesla devices to any provider they authorize.
 
-Clients establish a websocket connection to push configurable telemetry records. Telemetry provides clients with ack, error, or rate limit responses.
+Fleet Telemetry is meant to be the service implementation providers can run at the recieving hand. The service will handle connectivity with the vehicles and other devices, receive and store data they transmit. In other words, Fleet Telemetry is a simple, scalable, and secure data exchange service for devices. Once configured to do so, devices can establish a websocket connection to push configurable telemetry records. Fleet Telemetry provides clients with ack, error, or rate limit responses.
```

Comment:
removed some small grammar issues, and unnecessary words:
```
Fleet Telemetry is a server reference implementation. The service handles vehicle/device connectivity and receives and stores transmitted data. Once configured, devices establish a websocket connection to push configurable telemetry records. Fleet Telemetry provides clients with ack, error, or rate limit responses.
```

I like this line in the first paragraph instead, or can remove: `Fleet Telemetry is a simple, scalable, and secure data exchange service for devices.`